src/js/disability-benefits/686/tests/config/applicantInfo.unit.spec.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { DefinitionTester, getFormDOM, submitForm } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import formConfig from '../../config/form';
+import ReactTestUtils from 'react-dom/test-utils';
```

Comment:
This is not a fun change, but we try to write new tests with Enzyme, instead of ReactTestUtils. It's pretty similar and the basic logic is the same, just some of the commands are different. You can look at any of the tests in `src/js/vre/tests/config` or `src/js/vic-v2/tests/config` to get examples.